guardian/typerighter
# Remove audit app

## What does this change?

Removes the audit app, introduced as a little experiment way back in  https://github.com/guardian/typerighter/pull/44.

We've got a story for testing rules pending in the rule manager client (e.g. #389), so this code is no longer needed, and removing it will speed up CI and reduce our dependency surface area.

## How to test

- The app should build and run as before
- There should be no 'audit' link in the header of the checker UI:

<img width="786" alt="Screenshot 2023-09-18 at 10 47 37" src="https://github.com/guardian/typerighter/assets/7767575/86e364bd-ef2c-4c65-80b6-1fd7a4d15453">
